modules/rest_api/controllers/services/rest.php

```diff
@@ -1276,7 +1317,16 @@ private function getEsActualUrl($url) {
    */
   private function getEsOutputHeader($format) {
     if ($format === 'csv') {
-      $csvTemplate = $this->getEsCsvTemplate();
+      // CSV template fields
+      $csvTemplate = array_values($this->getEsCsvTemplate());
+      // Survey specific sample fields
+      if (isset($this->esCsvSurveySampleColumns)) {
+        $csvTemplate = array_merge($csvTemplate, array_values($this->esCsvSurveySampleColumns));
```

Comment:
I'm not sure I understand why array_values is needed here - I might just be missing something?